modules/apps/transfer/module.go

```diff
@@ -109,7 +109,7 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	}
 
 	if err := cfg.RegisterMigration(types.ModuleName, 2, m.MigrateTotalEscrowForDenom); err != nil {
-		panic(fmt.Sprintf("failed to migrate total escrow amount from version 2 to 3: %v", err))
+		panic(fmt.Sprintf("failed to migrate transfer app from version 2 to 3: %v", err))
 	}
 
 	if err := cfg.RegisterMigration(types.ModuleName, 3, m.MigrateParams); err != nil {
```

Comment:
I will update also the error text for this one in a separate PR. Didn't do it here to make backport easier.

Reply:
doesn't it make more sense to be specific about which migration failed?